### PR TITLE
static videos-page

### PR DESCRIPTION
**Pull request by harshkotkar5557**
https://hareshkotkar.tinytake.com/msc/Njc2NjExMV8xOTM4NTA3Nw

### CODE REVIEW

**Review by Mahi24-prog — APPROVED** [review]
Looks good to merge

File: src/components/videoCard/index.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      <abbr title="Add to playlist">
+          <i
+            onClick={() => handlePlayListModal(video._id)}
+            className="fa fa-list list-icon"
+            aria-hidden="true"
+          ></i>
+        </abbr>
+        <div onClick={() => navigator(`/video/${video._id}`)}>
+          <img src={video.staticImg} alt={video.title} />
+
```
Comment (prathmeshjagtap) [inline src/components/videoCard/index.js:44]: you can also use link to get images with respect to id 
https://img.youtube.com/vi/<>video id here</>/maxresdefault.jpg

**Review by prathmeshjagtap — APPROVED** [review]
Code is Well structured
Naming Conventions are also good 
I liked your Protected route implementation

File: src/components/videoCard/index.js (hunk 1 above)
Comment (harshkotkar5557) [reply src/components/videoCard/index.js:44]: okay Thanks bro.

**Pull request merged**